addon-resizer/README.md

```diff
@@ -28,6 +28,7 @@ Usage of pod_nanny:
       --storage="MISSING": The base storage resource requirement.
       --threshold=0: A number between 0-100. The dependent's resources are rewritten when they deviate from expected by more than threshold.
       --use-metrics=false: Whether to use apiserver metrics to detect cluster size instead of the default method of listing node objects from the Kubernetes API.
+      --ignoreResourceRequests=false: ignoring scaling up or down of the resource requests if this flag is true.
```

Comment:
It is hard to tell what this flag actually does when specified. Expand the description and/or change the flag name (for example scaleResourceRequest with the default set to true would be much more intuitive).